File: Arrays/1635-number-of-good-pairs/number-of-good-pairs.kt
class Solution {
    fun numIdenticalPairs(nums: IntArray): Int {
        var pos = 0

        for (i in 0 until nums.size){
            for(j in i+1 until nums.size){
               if(nums[i] == nums[j]){
                   pos++
               }
            }
        }
        return pos
    }
}

File: TwoPointer/1635-number-of-good-pairs/number-of-good-pairs.swift
class Solution {
    func numIdenticalPairs(_ nums: [Int]) -> Int {
        var ans = 0

        for i in 0..<nums.count{
            for j in (i+1)..<nums.count{
                if(nums[i] == nums[j]){
                    ans += 1
                }
            }
        }
        return ans
    }
}